storage/gcp/gcp.go

```diff
@@ -145,16 +145,12 @@ func New(ctx context.Context, cfg Config, opts ...func(*tessera.StorageOptions))
 				return
 			case <-t.C:
 			}
-			for {
-				cctx, cancel := context.WithTimeout(ctx, 10*time.Second)
-				defer cancel()
-				if more, err := r.sequencer.consumeEntries(cctx, DefaultIntegrationSizeLimit, r.integrate); err != nil {
-					klog.Errorf("integrate: %v", err)
-					break
-				} else if !more {
-					break
-				}
-				klog.V(1).Info("Quickloop")
+			// Don't quicklook for now, it causes issues updating checkpoint too frequently.
+			cctx, cancel := context.WithTimeout(ctx, 10*time.Second)
```

Comment:
Is `10*time.Second` configurable? Is this part going to be reworked later to optimize the checkpoint update frquency?

Reply:
Not currently, this is really just a "fuse" to blow if some call to GCP gets wedged for whatever reason; things will retry and presumably continue as normal once whatever network/service issue has been resolved. 
I can't really imagine it needing to be greater than 10s, but we can always add a config option later on if it becomes necessary.